### DISCUSSION TITLE
魏老师打算更新`Contribution.md`吗？

### DISCUSSION THREAD

**Discussion by scaomath** [weihuayi/fealpy#42] [General]
我用VEM的时候指回reference发现有很多`#TODO`，但是这些`#TODO`没有说明什么问题或者什么情况下会出问题。

我建议魏老师可以把那个`Projects`功能打开。

**Comment by weihuayi** [comment]
@scaomath 我打开了 project 功能，但我可能还要详细研究一下怎么用，才能充分发挥它的作用。另外，那个 Contribution.md 可以自动更新吗？还是必须手动来增加？

**Reply by scaomath** [reply]
Contribution还是需要自己写一个，比如
- 大的包有很多workflow要check所以有好多需要注意的，比如`flake8`或者`autopep`一类： https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md
- 也有这种比较简短的 https://github.com/google-research/long-range-arena/blob/main/CONTRIBUTING.md
一般可能就是说欢迎大家pull request，但是有1,2,3,4写code的规矩，参考`xyz.py`的minimal example。

Project主要是用来track progress的，之前改PyTorch的code就发现他们的project做的挺好：https://github.com/pytorch/pytorch/projects/25
因为project可以直接用add column把issue拉进去。

**Comment by weihuayi** [comment]
另外，非常欢迎曹老师深度参与 FEALPy 的开发工作啊！

**Reply by scaomath** [reply]
现在FEALPy的文档是在魏老师的个人网站下面，自己感觉建议可以尝试下直接用[`gh-pages`](https://github.com/weihuayi/fealpy/settings/pages)弄到`fealpy`的repo下面，放在docs文件夹就可以了。

这样每个人可以在自己的branch里面随便写，写完了弄个PR魏老师检查下就merge到master里面。

**Comment by weihuayi** [comment]
好的，我设置一下。

----------

该邮件从移动设备发送




--------------原始邮件--------------
发件人："Shuhao Cao ***@***.***&gt;;
发送时间：2021年7月13日(星期二) 凌晨2:45
收件人："weihuayi/fealpy" ***@***.***&gt;;
抄送："魏华祎 ***@***.***&gt;;"Comment ***@***.***&gt;;
主题：Re: [weihuayi/fealpy] 魏老师打算更新`Contribution.md`吗？ (#42)
-----------------------------------

 

 
现在FEALPy的文档是在魏老师的个人网站下面，自己感觉建议可以尝试下直接用gh-pages弄到fealpy的repo下面，放在docs文件夹就可以了。
 
这样每个人可以在自己的branch里面随便写，写完了弄个PR魏老师检查下就merge到master里面。

**Reply by weihuayi** [reply]
曹老师，我已经设置了 gh-pages, 现在 fealpy 下面有个 docs 文件夹。只是域名我还要重新设置一下，目前是 https://www.weihuayi.cn/fealpy 和我个人网站的设置冲突了。

**Reply by weihuayi** [reply]
另外，我还申请了 fealpy.org  的域名，只是国内 org 的域名认证不了，无法启用。
